UPGRADE.md

```diff
@@ -0,0 +1,80 @@
+UPGRADE GUIDE FROM 1.x to 2.0
+=============================
+
+This is guide for upgrade from version 1.x to 2.0 for using the CLI tool.
+
+Config file
+-----------
+From now you can create new configuration file: `.php_cs.dist`. This file is prioritized over `.php_cs` configuration file. It is recommended to use first one and add second one to `.gitignore` to allow your contributors to have theirs own configuration file.
+
+Config and Finder classes
+-------------------------
+All off `Symfony\CS\Config\*` and `Symfony\CS\Finder\*` classes have been removed, instead use `Symfony\CS\Config` and `Symfony\CS\Finder`.
+
+For that reason one can not set config class by `--config` CLI argument, from now it is used to set configuration file. Thanks to this the `--config-file` CLI argument is no longer available.
+
+Rules and sets
+--------------
+To configure which fixers should be used one must now set rules and sets instead of fixers and level. This affects both configuration file and CLI arguments.
+
+The term of risky rules was introduced. Risky rule is a rule that may change the meaning of code (like `strict` rule, which will change `==` into `===`). No risky rules are run be default. One need to explicitly permit risky rules to run them.
```

Comment:
Hmm, I guess there is a technical difference to a rule being risky, and our implementation of that rule, a "fixer" being risky.

For example, I'd argue the fixer we have that normalizes the case on "internal php function calls" should be marked as risky because the "fixer" is risky imo even though the "rule" is not.

The fixer is risky in the sense that it can make false changes, but not in the sense that it breaks the code, but given that that's true for all cs fixers, I strongly believe that this fixer is infact risky.